recipes/pretextsnapshot/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     machine=Linux;;
+    Darwin*)    machine=Mac;;
+esac
+
+mkdir -p $PREFIX/bin
+
+if [ $machine = "Linux" ]; then
+awk -v rep="-fno-builtin-exp -fno-builtin-log -fno-builtin-pow -lstdc++ -lm" '{ gsub(/\-lstdc\+\+ \-lm/, rep); print }' > _install < install
```

Comment:
BTW: `conda-build` also supports patch files for these kinds of things.
I only skimmed that install script, but it looks like one could do also `./install --flags="${CXXFLAGS} <additional flags>"`, too. But without more context (and time) I can't tell what the best solution is and let that decision up to you.